### PR TITLE
fix: Hide cluster stats when 0

### PR DESCRIPTION
**Pull request by gt2345**
## Description

Before: 
Screenshot 2023-11-07 at 3 21 03 PM

After:
Screenshot 2023-11-07 at 3 20 52 PM

## Test Plan

Log in to OSS or someone with `canAdministrateUsers` permission on EE
Check the clusters page after creating some tasks
Stats cards with 0 should not show up

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1616

### CODE REVIEW

File: webui/react/src/pages/Cluster/ClusterOverallStats.tsx
Comment (keita-determined) [inline webui/react/src/pages/Cluster/ClusterOverallStats.tsx:1]: question: 
is it expected or `Connected Agent` and `CUDA Slots Allocated` should be also hidden?

image

Comment (gt2345) [reply webui/react/src/pages/Cluster/ClusterOverallStats.tsx:1]: I don't think so, I think this only applies to what could be running on cluster?
@alexjdetermined

**Review by alexjdetermined — APPROVED** [review]
LGTM

**Review by keita-determined — APPROVED** [review]
LGTM

**Pull request merged**